documentation/start-documenting.rst

```diff
@@ -64,79 +64,109 @@ Building the documentation
 
-To build the documentation, follow the instructions from one of the sections
-below.  You can view the documentation after building the HTML by pointing
-a browser at the file :file:`Doc/build/html/index.html`.
+.. note::
 
-You are expected to have installed the latest stable version of
-Sphinx_ and blurb_ on your system or in a virtualenv_ (which can be
-created using ``make venv``), so that the Makefile can find the
-``sphinx-build`` command.  You can also specify the location of
-``sphinx-build`` with the ``SPHINXBUILD`` :command:`make` variable.
+   The following instructions all assume your current working dir is
+   the ``Doc`` subdirectory in your :ref:`CPython repository clone <checkout>`.
+   Make sure to switch to it with ``cd Doc`` if necessary.
 
 
-.. _building-using-make:
+.. _doc-create-venv:
+
+Create a virtual environment
+----------------------------
 
-Using make / make.bat
----------------------
+.. _doc-create-venv-unix:
 
-**On Unix**, run the following from the root of your :ref:`repository clone
-<checkout>` to build the output as HTML::
+**On Unix platforms** that support :program:`make`
+(including Linux, macOS and BSD),
+you can create a new :mod:`venv` with the required dependencies using::
 
-   cd Doc
    make venv
-   make html
 
-or alternatively ``make -C Doc/ venv html``.  ``htmlview`` can be used
-instead of ``html`` to conveniently open the docs in a browser once the
-build completes.
+Building the docs with :program:`make` will automatically use this environment
+without you having to activate it.
+
+.. _doc-create-venv-windows:
+
+**On Windows**, or if not using :program:`make`,
+`create a new virtual environment <venv-create_>`__ manually.
+Always be sure to `activate this environment <venv-activate_>`__
+before building the documentation.
+
+
+.. _building-using-make:
+.. _using-make-make-bat:
+.. _doc-build-make:
+
+Build using Make / make.bat
+---------------------------
 
-You can also use ``make help`` to see a list of targets supported by
-:command:`make`.  Note that ``make check`` is automatically run when
-you submit a :ref:`pull request <pullrequest>`, so you should make
-sure that it runs without errors.
+A Unix ``Makefile`` is provided, :cpy-file:`Doc/Makefile`,
+along with a :cpy-file:`Doc/make.bat` batch file for Windows
+that attempts to emulate it as closely as practical.
 
-**On Windows**, the :cpy-file:`Doc/make.bat` batchfile tries to emulate
-:command:`make` as closely as possible, but the venv target is not implemented,
-so you will probably want to make sure you are working in a virtual environment
-before proceeding, otherwise all dependencies will be automatically installed
-on your system.
+.. important::
 
-When ready, run the following from the root of your :ref:`repository clone
-<checkout>` to build the output as HTML::
+   The Windows ``make.bat`` batch file lacks a ``make venv`` target,
+   automatically installing any missing dependencies
+   into your current environment.
```

Comment:
```suggestion
   The Windows ``make.bat`` batch file lacks a ``make venv`` target
   to automatically install missing dependencies
   into your current environment.
```

Reply:
Actually, this intends to say the opposite, that instead of having a `venv` target for it, the `make.bat` always automatically installs the required dependencies into the current environment whenever it is run. However, I can definitely see how that is unclear, so here's some edits to clarify that:

```suggestion
   The Windows ``make.bat`` batch file lacks a ``make venv`` target.
   Instead, it automatically installs any missing dependencies
   into the currently activated environment (or the base Python, if none).
```